src/test/java/com/nhantran/base/TestBase.java

```diff
@@ -41,17 +32,17 @@ public TestBase() {
         }
     }
 
-    protected static void startAUT() {
+    protected void startAUT() {
         String browser = properties.getProperty("browser");
-        String url = properties.getProperty("url");
+        String url = Constants.RAILWAY_URL;
```

Comment:
railway url is defining 2 places: config.properties and constants????

Reply:
in the previous PR, I stored the railway url in the config.properties, but after the review, I stored it in the Constants.